aten/src/ATen/native/quantized/cpu/qconv_unpack.cpp

```diff
@@ -93,6 +105,10 @@ class QConvUnpackWeightsInt8 final : public c10::OperatorKernel {
 #endif
 #ifdef USE_PYTORCH_QNNPACK
     if (ctx.qEngine() == at::QEngine::QNNPACK) {
+      TORCH_CHECK(
+          kSpatialDim == 2,
+          "quantized::conv_unpack (qnnpack): QNNPACK only supports Conv2d "
+          "now.");
```

Comment:
:'( any plan to support it in the future?

Reply:
I guess there is? Actually I am not quite sure about that part.